test/tInstall.m

```diff
@@ -0,0 +1,41 @@
+classdef tInstall < matlab.unittest.TestCase
+% Verify installation of matlab engine
+
+% Copyright 2023 Mathworks, Inc.
+
+    properties (Constant)
+        MATLABVersion = string(ver('MATLAB').Version) % Example: 9.14
+        MATLABRelease = erase(ver('MATLAB').Release,{'(',')'}) % Example: R2023a
```

Comment:
Maybe change this:

% Example: R2023a

to this:

% Example: (R2023a) ->R2023a